src/dispatch/event/service.py

```diff
@@ -29,6 +30,11 @@ def get_by_incident_id(*, db_session, incident_id: int) -> List[Optional[Event]]
     return db_session.query(Event).filter(Event.incident_id == incident_id)
 
 
+def get_by_uuid(*, db_session, uuid: str) -> List[Optional[Event]]:
```

Comment:
nit

```suggestion
def get_by_uuid(*, db_session, uuid: str) -> list[Event | None]:
```

py3.9+ has `list` type builtin now and `Optional` is deprecated for `| None`